src/App.tsx

```diff
@@ -1,26 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+
+  const totalItems = items.length;
+  const firstItem = (currentPage - 1) * perPage;
+  const lastItem = firstItem + perPage > items.length
+    ? items.length
+    : firstItem + perPage;
+
+  const onSelect = (value: number) => {
```

Comment:
```suggestion
  const handleSelect = (value: number) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364